src/data-services/summary-stats.js

```diff
@@ -21,18 +21,20 @@ export const fetchSummaryStatistics = (
 
   // TODO: Remove fakery when backend available
   if (process.env.REACT_APP_SUMMARY_STATISTICS_URL === 'fake') {
-    return Promise.resolve(fake[variable]);
+    return new Promise(resolve => {
+      setTimeout(() => resolve(fake[variable]), 1000);
+    });
   }
   return axios.get(
-    urljoin(process.env.REACT_APP_SUMMARY_STATISTICS_URL, 'tbd'),
+    urljoin(process.env.REACT_APP_SUMMARY_STATISTICS_URL, 'percentileanomaly'),
```

Comment:
Given that we've already run in to error situations with the percentiles backend, I'd propose some handling of error conditions (400s, 500s) here or where ever is appropriate.

Reply:
Done